Fix pyproject.toml to also install iodata/formats, iodata/inputs and all from iodata/tests
Author: Pavel Tomanek (Inuits)
--- pyproject.toml.orig	2025-11-13 19:06:54.075481731 +0100
+++ pyproject.toml	2025-11-13 19:07:40.547905445 +0100
@@ -69,8 +69,8 @@
 addopts = "-n auto -W error --strict-markers"
 markers = ["slow: marks tests as slow (deselect with '-m \"not slow\"')"]
 
-[tool.setuptools]
-packages = ["iodata"]
+[tool.setuptools.packages.find]
+include = ["iodata*"]
 
 [tool.setuptools_scm]
 write_to = "iodata/_version.py"
